doc/template.html

```diff
@@ -67,5 +67,36 @@ <h3 class="date">$date$</h3>
 $for(include-after)$
 $include-after$
 $endfor$
+<script type="text/javascript">
+  (function() {
+    // write out a basic 'dropdown_link' reference element
+    var dropdown_link = document.createElement('a');
+    dropdown_link.className = 'dropdown_link';
+    dropdown_link.innerHTML = ' <b>+</b>'; // the space is for padding
```

Comment:
If the space is padding then you should use `&nbsp;`.
But, if it's just padding: why not use the css/style for it? i.e. `dropdown_link.style["padding-left"]`